CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## UNRELEASED
 * **Removed**
 * **Bug Fix**
+   * FIXED: All logging in `valhalla_export_edges` now goes to stderr [#4892](https://github.com/valhalla/valhalla/pull/4892)
```

Comment:
goes at the bottom

Reply:
Sorry, not quite sure I follow the change request. I _think_ I followed the same order as prior releases (Removed, Bug Fix, Enhancement)...

Reply:
sorry in my haste I didn't see the other section header!